### PR TITLE
Anonymous guid

### PR DESCRIPTION
**Pull request by ramico**
supported a solution  -don't know how

### CODE REVIEW

**Comment by ramico** [comment]
removing anchor tag  shall wait

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -144,6 +144,14 @@ private function output_settings_debug() {
 		}
 
 		private function output_settings_reporting() {
+			if ( isset( $_GET[ 'reset_requested' ] ) ) {
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:147]: `reset_requested` isn't clear. Reset what? I think a better name would be `reset_guid` or something like that.

Comment (ramico) [reply includes/class-wc-siftscience-admin.php:147]: ok

**Comment by ramico** [comment]
> 
> 
> Looks good. Just needs a better name for the GET parameter.

done

**Pull request merged**